### PR TITLE
Update tekton documentation

### PR DESCRIPTION
**Pull request by livespotty**
Updated two files on tekton tutorial with latest apache/camel-k:1.9.0-SNAPSHOT
docs/modules/ROOT/attachments/tekton/camel-k-pipeline-task-definition.yaml
docs/modules/ROOT/pages/tutorials/tekton/tekton.adoc

**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by oscerd** [comment]
We need to squash and merge

File: docs/modules/ROOT/attachments/tekton/camel-k-pipeline-task-definition.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ metadata:
 spec:
   steps:
     - name: install
-      image: apache/camel-k:1.0.0-M2
+      image: docker.io/apache/camel-k:1.9.0-SNAPSHOT
```
Comment (squakez) [inline docs/modules/ROOT/attachments/tekton/camel-k-pipeline-task-definition.yaml:38]: I think it should go against the latest stable `1.8.0` instead. That image was not released and will end up with pulling errors.

**Review by squakez — CHANGES_REQUESTED** [review]
We should reference a stable version.

File: docs/modules/ROOT/attachments/tekton/camel-k-pipeline-task-definition.yaml (hunk 1 above)
Comment (livespotty) [reply docs/modules/ROOT/attachments/tekton/camel-k-pipeline-task-definition.yaml]: I have modified to stable version 1.8.0

**Review by squakez — APPROVED** [review]

**Pull request merged**